src/nvim/ex_cmds.c

```diff
@@ -166,10 +167,10 @@ void ex_align(exarg_T *eap)
 {
   pos_T save_curpos;
   int len;
-  int indent = 0;
-  int new_indent;
+  long indent = 0;
+  long new_indent;
   int has_tab;
-  int width;
+  long width;
```

Comment:
You are not supposed to add `long` variables based on the style guide. You should try to change the other end of the error so that this change is not necessary.